src/arm.rs

```diff
@@ -1,62 +1,71 @@
 use core::intrinsics;
 
 #[cfg(feature = "mem")]
-use mem::{memcpy, memmove, memset};
+use mem::memmove;
 
-// NOTE This function and the ones below are implemented using assembly because they using a custom
-// calling convention which can't be implemented using a normal Rust function
+// NOTE This function and the ones below are implemented using assembly
+// because they using a custom calling convention which can't be implemented
+// using a normal Rust function
```

Comment:
I know it unrelated to this PR, but I couldn't help but notice these functions could be implemented using a normal rust function like
```
pub extern "C" fn __aeabi_uidivmod(n: u32, d: u32) -> u64 {
    let mut rem = 0;
    let quot = unsafe {__udivmodsi4(n, d, Option::Some(&mut rem))};
    ((rem as u64) << 32) | quot as u64
}
```
using a u64 as a 'in regs' struct.